tests/VAPIDTest.php

```diff
@@ -98,7 +98,7 @@ public function testCreateVapidKeys()
         $keys = VAPID::createVapidKeys();
         $this->assertArrayHasKey('publicKey', $keys);
         $this->assertArrayHasKey('privateKey', $keys);
-        $this->assertEquals(strlen($keys['publicKey']), 88);
-        $this->assertEquals(strlen($keys['privateKey']), 44);
+        $this->assertEquals(strlen($keys['publicKey']), 87);
```

Comment:
The condition becomes ">=86" and ">=42"

Reply:
fix the unit testing 
change assertEquasl to assertGreaterThanOrEqual